tree_sitter_v/test/corpus/struct_declaration.txt

```diff
@@ -447,8 +447,8 @@ Simple struct with fields with attributes and default values
 ================================================================================
 struct Foo {
     name  string [attr]
-    age   int    [attr] = 10
-    other f32    [attr; omitempty] = 3.14
```

Comment:
these are valid for the old attributes (before `@[attr]`)

Reply:
the new `@[attr]` is always last however

Reply:
will v-analyzer be able to parse existing code, that used the old style [attributes] after this change?

Reply:
The main V repo is now using the new `@[abc]` attributes, but I think it will be good if the old ones are supported for 1-2 months more, until people can have time to update their code with `v fmt -w .`.

Reply:
Making v-analyzer suggest using old style [attr] at the end of field declarations will be confusing.